lib/capybara-screenshot/cucumber.rb

```diff
@@ -18,8 +19,10 @@
         require "base64"
         #encode the image into it's base64 representation
         image = open(saver.screenshot_path, 'rb') {|io|io.read}
-        encoded_img = Base64.encode64(image)
+        #print image to screen, if imgcat is available
+        system("imgcat #{saver.screenshot_path}") if find_executable 'imgcat'
```

Comment:
What I don't like is `checking for imgcat... no` is now sent to STDOUT every time this block is run.  Could you perhaps do two things:

* Look at using a different way to detect whether `imgcat` is present using something like http://stackoverflow.com/a/5471032/139607
* Memoize whether `imgcat` is installed so we don't have to check for its existence every time